static/js/containers/VideoDetailPage.js

```diff
@@ -147,30 +151,38 @@ class VideoDetailPage extends React.Component {
               <h2 className="video-title mdc-typography--title">
                 {video.title}
               </h2>
-              { video.description &&
-              <p className="video-description mdc-typography--body1">
-                {video.description}
-              </p>
-              }
-              <div className="upload-date mdc-typography--subheading1 fontgray">
-                Uploaded {formattedCreation}
-              </div>
               <div className="actions">
-                {
-                  editable &&
-                  <Button
-                    className="edit mdc-button--raised"
-                    onClick={showDialog.bind(this, DIALOGS.EDIT_VIDEO)}
-                  >
-                    <span className="material-icons">edit</span> Edit
-                  </Button>
-                }
                 <Button
                   className="share mdc-button--raised"
                   onClick={showDialog.bind(this, DIALOGS.SHARE_VIDEO)}
                 >
                   <span className="material-icons ">share</span> Share
                 </Button>
+                {
+                  editable &&
+                  <span>
+                    <Button
+                      className="edit mdc-button--raised"
+                      onClick={showDialog.bind(this, DIALOGS.EDIT_VIDEO)}
+                    >
+                      <span className="material-icons">edit</span> Edit
+                    </Button>
+                    <Button
+                      className="download mdc-button--raised"
```

Comment:
class name should probably be changed ("download" class name vs. "Save to Dropbox" label)

Reply:
OK